packages/gestalt/src/Masonry.tsx

```diff
@@ -120,14 +120,28 @@ type Props<T> = {
    *
    * This is an experimental prop and may be removed or changed in the future.
    */
-  _getColumnSpanConfig?: (item: T) => ColumnSpanConfig;
+  _getColumnSpanConfig?: (item: T | U) => ColumnSpanConfig;
+  /**
+   * An array of items to display that contains the data to be rendered by `_renderLoadingStateItems`.
+   */
+  _loadingStateItems?: ReadonlyArray<U>;
```

Comment:
do we need a generic type for `_loadingStateItems` or can we just be specific? aside from `height` being passed in, is there anything else we need?

Reply:
good callout, i dont think this needs to be generic since we will be the ones controlling this in webapp and we dont need anything besides `height` (makes the types easier too!)